spec/features/admin_can_edit_a_place.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+feature "Admin can edit a place they created and a place they did not create", %Q{
+  As an admin
+  I want to edit the place I created and places others created
+  So that I can oversee the info on the places pages that are on the website
+} do
+
+  let!(:user) {User.create!(
+    id: 1,
+    first_name: "Jose",
+    last_name: "Fine",
+    email: "jf@hotmail.com",
+    password: "welcome1",
+    role: "member"
+  )}
+  let!(:user2) {User.create!(
+    id: 2,
+    first_name: "Amy",
+    last_name: "Fine",
+    email: "af@hotmail.com",
+    password: "password",
+    role: "member"
+  )}
+
+  let!(:user3) {User.create!(
+    id: 3,
+    first_name: "Chris",
+    last_name: "Fine",
+    email: "Cf@hotmail.com",
+    password: "password",
+    role: "admin"
+  )}
+
+  scenario 'admin is able to edit a place they did not create' do
+    place1 = Place.create!(user: user, name: "Forge",place_type: "Coffee Shop",neighborhood: "Union Square",description: "spacious bakery that serves Intelligensia coffee and ice cream",wifi: true,food: true,outdoor_seating: false,standing_options: true,group_capacity: 4.0,address: "626 Somerville Ave",city: "Somerville",state: "MA",zip: "02143")
+
+    visit new_user_session_path
+    fill_in 'Email', with: user3.email
+    fill_in 'Password', with: user3.password
+    click_button 'Log in'
+
+    visit "/places/#{place1.id}"
+    expect(page).to have_content("Edit")
+  end
+  scenario 'admin is able to edit a place they did create' do
```

Comment:
Please add a space here, after line 16, and delete line 57! Prettiness counts :)

Reply:
er, space == empty line